test/fluree/server/integration/basic_query_test.clj

```diff
@@ -1,17 +1,17 @@
 (ns fluree.server.integration.basic-query-test
   (:require [clojure.test :refer [deftest is testing use-fixtures]]
+            [fluree.db.util.json :as json]
             [fluree.server.integration.test-system
              :as test-system
-             :refer [api-post create-rand-ledger json-headers run-test-server]]
-            [jsonista.core :as json]))
+             :refer [api-post create-rand-ledger json-headers run-test-server]]))
 
 (use-fixtures :once run-test-server)
 
 (deftest ^:integration ^:json query-json-test
   (testing "can query a basic entity w/ JSON"
     (let [ledger-name (create-rand-ledger "query-endpoint-basic-entity-test")
           txn-req     {:body
-                       (json/write-value-as-string
```

Comment:
Is there a reason we're switching away from [jsonista](https://github.com/metosin/jsonista) here? It's [faster](https://www.metosin.fi/blog/faster-json-processing-with-jsonista) than [cheshire](https://github.com/dakrone/cheshire), which is what we're wrapping in `fluree.db.util.json`.

Reply:
we don't have any explicit dependency on jsonista; we only implicitly bring it in because we have a dependency on malli. furthermore, we already have our own json parsing/encoding namespace, so we should always use that. if jsonista is truly better than cheshire, then we should explicitly depend on it and change the implementation of our json encoding/decoding functions to use it. that way we reap those benefits everywhere.